components/dashboard/script.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+
+const vars = {
+    REACT_APP_HOSTNAME : 'Gitpod',
+    REACT_APP_DOCS_URL : 'https://www.gitpod.io/docs/',
+    REACT_APP_SOCIAL_URL : 'https://community.gitpod.io/',
+    REACT_APP_FAVICON_192 : './favicon192.png',
+    REACT_APP_FAVICON_256 : './favicon256.png',
+    REACT_APP_FAVICON_512 : './favicon512.png'
+};
+
+if(!fs.existsSync('.env')) {
```

Comment:
Why do we need .env file ? 

Check if process.ENV has a value if so override var[XYZ]=process.ENV_XYZ

Reply:
Checking the process.ENV value is not so elegant looking solution. Ideally, we should be using just the env variable everywhere but due to the reason mentioned [here](https://github.com/trilogy-group/gitpod/pull/18/files#r666762327) that is not possible for some icons as they are not kept in the public folder.